### PR TITLE
Add custom tabs for mobile devices to match the design

### PR DESCRIPTION
**Pull request by Pallavi-PH**
Signed-off-by: Pallavi-PH <pallaviph02@gmail.com>

In this PR we are adding custom tabs for mobile devices to match the design

Tabs on Home page:
Screenshot 2021-07-15 at 12 48 57 PM

Tabs on Blog page:
Screenshot 2021-07-15 at 12 38 56 PM

### CODE REVIEW

File: website/src/pages/Blog/index.tsx [hunk 1]
```diff
@@ -161,16 +161,24 @@ const Blog: React.FC = () => {
...
-          key={item}
-        />
+        mediumViewport ? 
+          <StyledTab
+            label={`${item} (${tagsDistribution[item as keyof typeof tagsDistribution]})`}
+            value={item}
+            key = {item}
+          />
+          :
+          <Grid item xs={6} key={item}>
```
Comment (AVRahul) [inline website/src/pages/Blog/index.tsx:179]: IMO, one line comment would be good to have here, rest changes looks good  to me.

File: website/src/pages/Blog/index.tsx [hunk 2]
```diff
@@ -161,16 +161,24 @@ const Blog: React.FC = () => {
     );
   };
 
+  const handleTabClickOnMobile = (newValue: string) => {
+    setValue(newValue);
+    setPage(1);
+  }
```
Comment (rakeshPRaghu) [inline website/src/pages/Blog/index.tsx:167]: why this function? Its same as `handleTagSelect`

File: website/src/pages/Blog/index.tsx (hunk 1 above)
Comment (Pallavi-PH) [reply website/src/pages/Blog/index.tsx:175]: Done

File: website/src/pages/Blog/index.tsx (hunk 2 above)
Comment (Pallavi-PH) [reply website/src/pages/Blog/index.tsx]: Removed this function

File: website/src/components/MiniBlog/index.tsx [hunk 3]
```diff
@@ -188,8 +186,8 @@ const MiniBlog: React.FC = () => {
               orientation={mobileBreakpoint ? "horizontal" : "vertical"}
             >
               <StyledTab
-                label={"All(" + totalBlogCount + ")"}
-                value={t("blog.all")}
+                label={t('blog.all') + " (" + totalBlogCount + ")"}
```
Comment (satyapriyamishra222) [inline website/src/components/MiniBlog/index.tsx:189]: Can we try with string interpolation here ?

File: website/src/pages/Blog/index.tsx [hunk 4]
```diff
@@ -163,14 +163,18 @@ const Blog: React.FC = () => {
...
-        />
+        mediumViewport ? 
+          <StyledTab
+            label={`${item} (${tagsDistribution[item as keyof typeof tagsDistribution]})`}
+            value={item}
+            key = {item}
+          />
+          :
+          <Grid item xs={6} key={item}>
+            <Button className={[classes.tabButton, (value === item) ? classes.activeTabButton : ''].join(' ')} onClick={() => handleTagSelect(item)}>{item} <span className={(value !== item) ? classes.tagCount : ''}>({tagsDistribution[item as keyof typeof tagsDistribution]})</span></Button>
```
Comment (satyapriyamishra222) [inline website/src/pages/Blog/index.tsx:176]: Let us please make this more readable by using proper indentation.

File: website/src/pages/Blog/index.tsx [hunk 5]
```diff
@@ -184,26 +188,36 @@ const Blog: React.FC = () => {
...
+                  root: classes.tabRoot, scroller: classes.scroller }}
+                TabIndicatorProps={{
+                  style: {
+                    display: "none",
+                  },
+                }}
+                orientation="horizontal"
+              >
+                <StyledTab
+                  label={t('blog.all') + " (" + totalBlogCount + ")"}
```
Comment (satyapriyamishra222) [inline website/src/pages/Blog/index.tsx:207]: Can we try string interpolation ?

File: website/src/pages/Blog/index.tsx [hunk 6]
```diff
@@ -184,26 +188,36 @@ const Blog: React.FC = () => {
...
+                  value={t("blog.all").toLowerCase()}
+                />
+                {getTagsMarkup}
+              </Tabs>
+              :
+              <Grid container className={classes.mobileTabsWrapper}>
+                  <Grid item xs={6}>
+                      <Button className={[classes.tabButton, (value === t("blog.all").toLowerCase()) ? classes.activeTabButton : ''].join(' ')} onClick={() => handleTagSelect(t("blog.all").toLowerCase())}>{t('blog.all')} <span className={(value !== 'all') ? classes.tagCount : ''}>({totalBlogCount})</span></Button>
```
Comment (satyapriyamishra222) [inline website/src/pages/Blog/index.tsx:215]: Let's make it more readable through proper indentation.

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by IsAmrish — APPROVED** [review]
lgtm

File: website/src/components/MiniBlog/index.tsx [hunk 7]
```diff
@@ -111,7 +111,7 @@ const MiniBlog: React.FC = () => {
   ).length;
 
   const filteredData = (jsonMdData || []).filter((tabs: TabProps) => {
-    if (value !== "all") {
+    if (value !== t("blog.all").toLowerCase()) {
```
Comment (rakeshPRaghu) [inline website/src/components/MiniBlog/index.tsx:114]: taking this value from translation is not a good idea

Comment (Pallavi-PH) [reply website/src/components/MiniBlog/index.tsx:114]: Can you please brief me about why we need to use a string here instead of translation?

Comment (rakeshPRaghu) [reply website/src/components/MiniBlog/index.tsx:114]: The idea of adding translation is for adding it in different languages, in this case it should come from a constant.

**Review by rakeshPRaghu — APPROVED** [review]
LGTM

**Review by AVRahul — APPROVED** [review]

**Pull request merged**